packages/adapters-library/src/adapters/lido/tests/snapshots/ethereum.tvl.wst-eth.json

```diff
@@ -14,30 +14,7 @@
         "type": "StandardErc20"
       },
       "success": true,
-      "tokens": [
-        {
-          "address": "0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84",
-          "name": "Liquid staked Ether 2.0",
-          "symbol": "stETH",
-          "decimals": 18,
-          "type": "protocol",
-          "totalSupplyRaw": "9391867043243646285936132n",
-          "tokens": [
-            {
-              "address": "0x0000000000000000000000000000000000000000",
-              "symbol": "ETH",
-              "name": "Ethereum",
-              "decimals": 18,
-              "totalSupplyRaw": "7177864073643285936132n",
-              "type": "underlying",
-              "totalSupply": 7177.864073643286,
-              "price": 3148.07,
-              "iconUrl": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/info/logo.png"
-            }
-          ],
-          "totalSupply": 9391867.043243647
-        }
-      ]
+      "tokens": []
```

Comment:
same here

Reply:
Those are expected as I'm passing a filter. The first one filters by st-eth and the second by wst-eth.

Having said that, I should really only have one test in which I pass both addresses, so I'll change that.

Reply:
Sorted